frontend/mr-admin-flate/src/layouts/ContainerLayout.module.scss

```diff
@@ -1,3 +1,18 @@
-.container_layout {
-  padding: 0.5rem;
+.container_layout_oversikt {
+  padding: 1rem;
+  max-width: 1920px;
+}
+
+.container_layout_detaljer {
+  padding: 1rem;
+  max-width: 1920px;
+  margin: 0 auto;
+
+  @media (width > 1920px) {
```

Comment:
Her har vi 1920px. I en annen fil har vi 1930px. Hvorfor ulikt?

Reply:
Det er for å unngå en liten hopping som skjer i komponenten når skjermen blir mindre.